modules/ROOT/pages/microsoft/microsoft-365-ops-connector-reference.adoc

```diff
@@ -3,327 +3,317 @@ ifndef::env-site,env-github[]
-| Streaming Strategy a| * repeatable-in-memory-iterable
-* repeatable-file-store-iterable
-* non-repeatable-iterable |  Configure to use repeatable streams and their behavior. |  |
-| Target Variable a| String |  The name of a variable that stores the output of the operation. |  |
-| Target Value a| String |  An expression that evaluates the operation's output and stores the outcome of that expression in the target variable. |  `#[payload]` |
-| Reconnection Strategy a| * reconnect
-* reconnect-forever |  A retry strategy to use for connectivity errors. |  |
+| Configuration | String | The name of the configuration to use. | | x
+| Datasense Query a| String |  The DSQL query that is going to use for retrieve. The query is transformed into a URL internally. |  #[payload] | 
+| Streaming Strategy a| * <<repeatable-in-memory-iterable>>
+* <<repeatable-file-store-iterable>>
+* non-repeatable-iterable |  Configure to use repeatable streams. |  | 
+| Target Variable a| String |  The name of a variable to store the operation's output. |  | 
+| Target Value a| String |  An expression to evaluate against the operation's output and store the expression outcome in the target variable. |  #[payload] | 
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors |  | 
 |===
 
-=== Output
-
-[cols="50a,50a"]
+==== Output
+[%autowidth.spread]
 |===
-| Type | Array of Object
+|Type |Array of Object
 |===
 
-=== For Configurations
-
-* dynamics-365-for-operations
+==== For Configurations
+* <<dynamics-365-for-operations>> 
 
-=== Throws
+==== Throws
 
-* DYNAMICS365FOROPERATIONS:UNKNOWN
-* DYNAMICS365FOROPERATIONS:INVALID_CREDENTIALS
-* DYNAMICS365FOROPERATIONS:INVALID_CONNECTION
-* DYNAMICS365FOROPERATIONS:LOGIN_FAILED
-* DYNAMICS365FOROPERATIONS:CONNECTIVITY
+* DYNAMICS365FOROPERATIONS:CONNECTIVITY 
+* DYNAMICS365FOROPERATIONS:INVALID_CONNECTION 
+* DYNAMICS365FOROPERATIONS:INVALID_CREDENTIALS 
+* DYNAMICS365FOROPERATIONS:LOGIN_FAILED 
+* DYNAMICS365FOROPERATIONS:UNKNOWN 
 
 
 [[unauthorize]]
-== Unauthorize
-
+=== Unauthorize
 `<dynamics365ForOperations:unauthorize>`
 
-Deletes all access token information for a resource owner ID so that it's impossible to
-execute any operation for that user without doing the authorization dance again.
 
-=== Parameters
+Deletes all the access token information of a given resource owner ID so that it's impossible to execute any operation for that user without doing the authorization dance again.
+
 
-[%header%autowidth.spread]
+==== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
 |===
 | Name | Type | Description | Default Value | Required
-| Configuration | String | The name of the configuration to use. | |x
-| Resource Owner Id a| String |  The ID of the resource owner to invalidate. |  |
+| Configuration | String | The name of the configuration to use. | | x
+| Resource Owner Id a| String |  The ID of the resource owner which access should be invalidated. |  | 
```

Comment:
The ID of the resource owner for which access should be invalidated?? 
I don't understand it as written.